docs/my-website/sidebars.js

```diff
@@ -215,6 +215,7 @@ const sidebars = {
       items: [
         "embedding/supported_embedding",
         "image_generation",
+        "image_analysis",
```

Comment:
Hi @krrishdholakia ,

Thank you for your valuable feedback on my contribution. Upon reviewing further, I realize that adding a separate "Image Analysis" section in Supported Endpoints would indeed duplicate the existing Vision section already documented under Chat Completion.

I've noticed that the current Vision documentation primarily focuses on examples using the OpenAI and LiteLLM Python SDKs. My intention was to add examples that leverage the REST API directly, which could be beneficial for users who prefer working with the REST interface rather than SDKs.

Would it be acceptable if I integrate my REST examples into the existing Vision documentation instead? This would provide more variety, like structured data extraction from images, while keeping the documentation more organized and avoiding any duplication in the Supported Endpoints section.

Looking forward to your thoughts!

Reply:
sure - feel free to update your pr to focus on that instead.